tools/secret-manager-helper/.gitignore

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Google Inc.
```

Comment:
Copyright 2021

Reply:
Thank you, fixed in bf9600a74b2db49037136c25ce4b9b9553467fdd.